easybuild/easyconfigs/u/UCX/UCX-1.11.2-missing_header.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/ucx-1.11.2-orig/src/uct/ib/base/ib_device.h b/ucx-1.11.2/src/uct/ib/base/ib_device.h
```

Comment:
Please include a short description of why the change is here and who the author is (you)

Reply:
Done. Sorry for that and thanks for spotting!